pkg/cmd/server/start/start_node.go

```diff
@@ -280,8 +274,13 @@ func StartNode(nodeConfig configapi.NodeConfig) error {
 
 	config.EnsureVolumeDir()
 	config.EnsureDocker(docker.NewHelper())
+	plugin, endpointFilter := InitSDNPlugin(config, nodeConfig)
```

Comment:
I don't understand... it's already in the `cmd/server/kubernetes` package... I just want to make it a field in cmd/server/kubernetes/NodeConfig... why does it need to move?